## Домашнее работа [1]

Работа с pandas. Необходимо ответить на вопросы, пользоваться можно только теми библиотеками, которые импортированны в 1 ячейке. После того как вы получите ответы на поставленные вопросы, нужно будет заполнить [Google форму](https://forms.gle/tDvH5evuzLVwZrHs5).

In [1]:
import pandas as pd
import numpy as np 

In [2]:
data = pd.read_csv('world-happiness/2015.csv')

In [3]:
data.head()

Country          Region  Happiness Rank  Happiness Score  \
0  Switzerland  Western Europe               1            7.587   
1      Iceland  Western Europe               2            7.561   
2      Denmark  Western Europe               3            7.527   
3       Norway  Western Europe               4            7.522   
4       Canada   North America               5            7.427   

   Standard Error  Economy (GDP per Capita)   Family  \
0         0.03411                   1.39651  1.34951   
1         0.04884                   1.30232  1.40223   
2         0.03328                   1.32548  1.36058   
3         0.03880                   1.45900  1.33095   
4         0.03553                   1.32629  1.32261   

   Health (Life Expectancy)  Freedom  Trust (Government Corruption)  \
0                   0.94143  0.66557                        0.41978   
1                   0.94784  0.62877                        0.14145   
2                   0.87464  0.64938                        0.48357   
3                   0.88521  0.66973                        0.36503   
4                   0.90563  0.63297                        0.32957   

   Generosity  Dystopia Residual  
0     0.29678            2.51738  
1     0.43630            2.70201  
2     0.34139            2.49204  
3     0.34699            2.46531  
4     0.45811            2.45176

In [4]:
data.shape

(158, 12)

__1. Сколько памяти необходимо для работы с датасетом ?__

In [ ]:
#Your code here

__2. Есть ли страны у которых Happiness Rank одинаковый ?__

In [ ]:
#Your code here

__3. Каково среднее значение (mean) и СКО (std) параметра Family ?__

In [ ]:
#Your code here

__4. Укажите 3 страны с самым низким уровнем параметра Trust (Government Corruption)__

In [ ]:
#Your code here

__5.Каково среднее значение (mean) для  стран с  Happiness Rank меньше или равным 10?__

In [ ]:
#Your code here

__6. Каково среднее значение (mean) для  стран принадлежащих к Western Europe и Economy (GDP per Capita) лежащим в интервале (1.35, 1.5) ?__

In [ ]:
#Your code here

__7. Какова сумма параметра Generosity для стран, принадлежащих к Sub Sahara Africa, чей Happiness Score болше 5 ?__

In [ ]:
#Your code here

__8. Действительно ли, что среднее значение Happiness Score у 5 стран с самым большим Happiness Score и принадлежащим к Southeastern Asia, будет больше чем у худших 5 стран по Happiness Score, но принадлежащим к Western Europe ?__

In [ ]:
#Your code here

__9. Какое значение примет параметр Standard Error если возвести его в куб и просуммировать, для трёх стран с самым маленьким Dystopia Residual, принадлежащим к Southern Asia ?__

In [5]:
#Подсказка: используйте pandas.DataFrame.agg() и лямбда функцию
#Your code here